### ISSUE TITLE
Osqueryd will not run

### ISSUE THREAD

**Issue by tnelson01** [osquery/osquery#6261]
# Bug report

### What operating system and version are you using?

 version = 10.0.18362
   build = 18362
platform = windows

### What version of osquery are you using?

4.1.2
### What steps did you take to reproduce the issue?

Used command:  osqueryd.exe  --verbose --flagfile="C:\Program Files\osquery\osquery.flags"

### What did you expect to see?

See trail of actions that led to osquery daemon running

### What did you see instead?

I0220 14:38:39.172595   912 init.cpp:418] osquery initialized [version=4.1.2]
I0220 14:38:39.202530   912 system.cpp:330] Found stale process for osqueryd (3748)
I0220 14:38:39.202530   912 system.cpp:362] Writing osqueryd pid (4452) to \Program Files\osquery\osqueryd.pidfile
I0220 14:38:39.202530   912 extensions.cpp:349] Could not autoload extensions: Failed reading: \Program Files\osquery\extensions.load

It's pointing to the "extensions.load" file that should be in the "osquery" folder, but this file wasn't installed when I installed osquery from the latest MSI downloaded from the osquery site (https://osquery.io/downloads/official/4.1.2).  Osqueryi runs fine.  Should I reinstall osquery?

**Comment by Smjert** [comment]
Hi @tnelson01!

How have you verified that osqueryd doesn't run?
That message you're referring to is not really an error/fatal error that would not permit osquery to run, it's just that by default osquery searches for that file to autoload extensions, and if it doesn't find it, it lets you know about it.

Moreover that file was never being installed by the MSI installer, the user has to create it.

**Comment by muffins** [comment]
@tnelson01 building on top of what @Smjert already eluded to, it looks like osqueryd _is_ running based off of that output. What are you expecting that you aren't seeing? By default osquery runs in a local filesystem mode, meaning its reading files from the filesystem to retrieve its configuration, and subsequently writing results out to the `C:\Program Files\osquery\log` folder by default. Mind taking a peak in the afore mentioned folder to see if there's results files?

**Comment by tnelson01** [comment]
Thank @muffins and @Smjert for the additional information, although it didn't really resolve the issue. The issue that was occurring was that osqueryd did initially attempt to start (hence the print out I provided) but then it would stop.  The daemon wouldn't stay running.  When the daemon started looking for the "extensions.load" file it would quit.  

The way I verified it wasn't running in the background was reviewing the detailed view of processes running on the box via the WindowsTask Manager.  When "osqueryd.exe" is running its there in the list.   I also double checked this with using the command "sc.exe query osqueryd".

As requested I attempted to contact the osquery team via Slack about this, but never received a response.  This was my next best avenue in order to try to receive feedback for this issue.  Luckily, after some more trial and error troubleshooting I was successful in having the daemon running in the background.

**Comment by Smjert** [comment]
> As requested I attempted to contact the osquery team via Slack about this, but never received a response. This was my next best avenue in order to try to receive feedback for this issue. Luckily, after some more trial and error troubleshooting I was successful in having the daemon running in the background.

I see, I'm sorry if we didn't answer you on Slack, but this is also a valid way to get a hold of us in an async way :).
I'm curious, what was the issue then?

**Comment by tnelson01** [comment]
No problem, I'm sure you're bombarded with tons of questions every day. 

To be honest I'm not sure what exactly the issue was, but after some additional digging around on the Internet, I heard of the suggestion to delete the current log files in the "log" folder.  I did that and it seemed to resolve the issue.  The daemon kept running afterwards.  

Thank you again for looking into this, I appreciate it. 😁

**Comment by muffins** [comment]
Interesting. Glad you got it figured out. Were you installing via the stock MSI on the site? I'm curious if I can reproduce that issue..

**Comment by tnelson01** [comment]
Hi @muffins,
Yes I used the stock MSI (4.1.2) from the website.  To provide a little background about my specific project, I was downloading OSQuery to use in conjunction with a Fleet server.  I was using an article to walkthrough the process of installation, so after I installed OSquery I added the recommended flags to the flags file to make the connection.  The problem was is that initially the osquery daemon wasn't running, so I created a service to run it, and thats when I started to run into the issue I reported.
